File: python/6.py
sum_of_squares = 0
square_of_sum = 0

for i in range(1, 100):
    sum_of_squares += i * i
    
for j in range(1, 100):
    square_of_sum += j
    
square_of_sum = square_of_sum * square_of_sum

print(square_of_sum - sum_of_squares)

File: c++/6.cpp
#include <iostream>

using namespace std;

int main(){
    long sumOfSquare = 0;
    long squareOfSum = 0;
    for(int i = 1; i <= 100; i++){
        sumOfSquare += i * i;
        squareOfSum += i;
    }
    squareOfSum = squareOfSum * squareOfSum;
    cout << squareOfSum - sumOfSquare << endl;
    return 0;
}
